e2e/tests/stepDefinitions/manage/lostBed.ts

```diff
@@ -0,0 +1,56 @@
+import { Given, Then } from '@badeball/cypress-cucumber-preprocessor'
+import Page from '../../../../cypress_shared/pages/page'
+import BedspaceShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/bedspaceShow'
+import LostBedNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedNew'
+import newLostBedFactory from '../../../../server/testutils/factories/newLostBed'
+import lostBedFactory from '../../../../server/testutils/factories/lostBed'
+import LostBedShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/lostBedShow'
+
+Given('I mark a bedspace as void', () => {
```

Comment:
Sorry, realised when I was changing E2E tests myself I was talking nonsense about this! Think you had this right the first time as this doesn't actually mark the bedspace as void, it just goes to the page to do that. There's a rough convention of "I'm X-ing" meaning "I going to the page to do X but not actually doing X yet"

Reply:
aha ok! have reverted that one (but not the second, for the reasons you give)